src/pages/studies/Sidebar.module.scss

```diff
@@ -44,9 +59,12 @@
   }
 
   & :global(.ant-collapse),
-  :global(.ant-collapse-content),
   :global(.ant-collapse-item) {
     background-color: $gray-1;
     border: none;
   }
 }
+
+:export {
+  width: 420;
```

Comment:
Here, it's clear that width is related to the sidebar. However, I doubt it will be as obvious of a name when using this variable in the calling file.

Reply:
in the refactoring I removed the need for it. So i'll delete it.  good catch